Add account and addresses how-tos (#447)
iotaledger/iota-sdk: documentation/sdk/docs/how_tos/accounts_and_addresses/create_account.mdx

---
title: Create an Account
description: 'How to create an account to start sending value transactions to the Tangle.'
image: /img/logo/iota_mark_light.png
keywords:
- how to
- create account
- create wallet
- account manager
- multi account approach
- single account approach
- nodejs
- python
- java
- rust
---

import CodeBlock from '@theme/CodeBlock';
import Tabs from "@theme/Tabs";
import TabItem from "@theme/TabItem";
import RustCode from "!!raw-loader!../../../../../sdk/examples/how_tos/accounts_and_addresses/create_account.rs";
import NodejsCode from "!!raw-loader!../../../../../bindings/nodejs/examples/how_tos/accounts_and_addresses/create_account.ts";
import PythonCode from "!!raw-loader!../../../../../bindings/python/examples/how_tos/accounts_and_addresses/create_account.py";
import TipAccountApproaches from "../../_admonitions/_tip_account_approaches.md";

> The first thing you will need to do when you are getting started with the SDK is to create an account.

Reviewer: No :trollface:

Author: I just copied the old example :trollface: 
Suggest another text. I'm not a TW

> The first thing you will need to do when you are getting started with the SDK is to create an account.

Reviewer: I'm also not a TW, but here's my suggestion :trollface: 
```suggestion
```

Author: ```suggestion
This How-To will show you how to create an account.
```

I improved it :trollface:

<TipAccountApproaches/>

## Code Example

The following example will:

> 1. Create an account manager.

Reviewer: ```suggestion
1. Create a wallet.
```

2. Use it to store the [Stronghold](https://wiki.iota.org/stronghold.rs/welcome) mnemonic.

> 3. Create a wallet with the alias "Alice".

Reviewer: ```suggestion
3. Create an account with the alias "Alice".
```
